benchmarks/tensors/dibaryon/baryon_wrapper.cpp

```diff
@@ -211,6 +213,7 @@ int main(int, char **)
 				      exit(1);
 				  }
 
+#if 0
```

Comment:
We should enable these checks by default. Does the code work if these checks are enabled?

Reply:
The new code never materializes Q, O, and P fully, so the checking code doesn't make sense. Should I just remove them?

Reply:
Oh I see. Sure, then better remove it. Just make sure that we have correctness checks of all the outputs. This is the only way we can protect against future bugs similar to that that we had.